Kernel/Storage/StorageManagement.cpp

```diff
@@ -30,7 +30,8 @@
 namespace Kernel {
 
 static Singleton<StorageManagement> s_the;
-static Atomic<u32> s_device_minor_number;
+static Atomic<u32> s_storage_device_minor_number;
+static Atomic<u32> s_partition_device_minor_number;
```

Comment:
If you're using the `MinorNumber` type alias in the APIs now, these can also just be `Atomic<MinorNumber>`, right?

Reply:
The compiler doesn't accept `MinorNumber` type as it is not "pure" primitive type like `u32`, so sadly we can't do this.